src/TypeScript/packages.lock.json

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
This file along with the project were deleted. Clean up the PR and your local git repo from those remaining files.